templates/zerver/help/delete-a-stream.md

```diff
@@ -0,0 +1,22 @@
+# Delete a stream
+On Zulip, most of the stream administration is done on the subscription page. Nevertheless, that is not the case for deleting a stream.
```

Comment:
Can you word wrap your markdown lines, please?

Also, here, I think we can just be succinct and say "Only Zulip administrators may delete streams."